Lesson9/anna/Lesson9-aysel/index.html

```diff
@@ -0,0 +1,243 @@
+        />
+
+        <p class="publisher">
+          <a href="https://medium.com/@adri_zag">Adrien Zaganelli</a> ·
+          <button>Follow</button> <br />
+          Published in
+          <a href="https://medium.com/free-code-camp"
+            >We’ve moved to freeCodeCamp.org/news</a>
+          ·
+          <span>3 min read · Aug 12, 2018</span>
+        </p>
+        <p class="responsive-publisher">
+          <a href="https://medium.com/@adri_zag">Adrien Zaganelli</a> ·
+          <button>Follow</button> <br />
+          Published in
+          <a href="https://medium.com/free-code-camp"
+            >We’ve moved to freeCodeCamp.org/news</a>
+          <br>
+          <span>3 min read · Aug 12, 2018</span>
+        </p>
+      </div>
+    <hr> <!--they disappeared on the browser i couldnt solve the problem-->
+      <div class="bottom-information">
+        <div class="left-bottom-information">
+          <i class="fa-solid fa-hands-clapping"> 10 K </i>
+          <i class="fa-regular fa-comment"> 21 </i>
+        </div>
+        <div class="right-bottom-information">
+          <i class="fa-regular fa-bookmark"></i>
+          <i class="fa-regular fa-circle-play"></i>
+          <i class="fa-solid fa-arrow-up-right-from-square"></i>
+        </div>  
+    </div>
+    <hr>
+
+    <img class="clock" src="https://miro.medium.com/v2/resize:fit:2000/format:webp/1*aVzJTznRRfP1lM7AXe9yLw.jpeg" 
+    srcset="https://miro.medium.com/v2/resize:fit:640/format:webp/1*aVzJTznRRfP1lM7AXe9yLw.jpeg 640w, 
+    https://miro.medium.com/v2/resize:fit:720/format:webp/1*aVzJTznRRfP1lM7AXe9yLw.jpeg 720w, 
+    https://miro.medium.com/v2/resize:fit:750/format:webp/1*aVzJTznRRfP1lM7AXe9yLw.jpeg 750w, 
+    https://miro.medium.com/v2/resize:fit:786/format:webp/1*aVzJTznRRfP1lM7AXe9yLw.jpeg 786w, 
+    https://miro.medium.com/v2/resize:fit:828/format:webp/1*aVzJTznRRfP1lM7AXe9yLw.jpeg 828w,
+     https://miro.medium.com/v2/resize:fit:1100/format:webp/1*aVzJTznRRfP1lM7AXe9yLw.jpeg 1100w, 
+     https://miro.medium.com/v2/resize:fit:2000/format:webp/1*aVzJTznRRfP1lM7AXe9yLw.jpeg 2000w" 
+     alt="clock-pic"/>
+     <figcaption>Photo by Kevin Ku on Unsplash</figcaption>
+   
+      <p>
+        As a web developer, there is a high probability that you have
+        encountered the two enemies of this article: images and deadlines.
+        Sometimes, for some reasons, your images won’t fit the layout and you
+        don’t want to wrap your head around this for hours.
+      </p>
+      <p>
+        This situation has happened to me many times and I have learned from my
+        mistakes. No more black magic hacks — here are my five favorites
+        techniques to handle image resizing.
+      </p>
+      <h5>the “OMG I NEED THIS ASAP” way</h5>
+      <p>
+        It’s 5:00 pm on Friday, you have to finish this page, but the images
+        won’t fit the layout. It’s time to use your magic trick!
+      </p>
+      <div class="gray-box">
```

Comment:
Use 'pre' element to add a pre-formatted text to the page. I've showed an example in the last lesson.
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/pre